### PR TITLE
Add pixel set selection cart

### PR DESCRIPTION
**Pull request by jmaupetit**
## Sneak peek

screenshot-2018-1-30 pixel sets 1

### CODE REVIEW

File: apps/explorer/forms.py [hunk 1]
```diff
@@ -79,6 +79,10 @@ class PixelSetExportForm(forms.Form):
     )
 
 
+class PixelSetSelectForm(PixelSetExportForm):
```
Comment (willdurand) [inline apps/explorer/forms.py]: Why is it necessary?

File: apps/explorer/templates/explorer/pixelset_list.html [hunk 2]
```diff
@@ -18,6 +18,64 @@ <h1>
...
+      {% if selected_pixelsets.count %}
+      <span class="subheader">
+        ({{ selected_pixelsets|length }})
+      </span>
+      {% endif %}
+    </h4>
+
+    <p class="help-text">
+      {% blocktrans %}
+      When you select and save pixel sets for exportation, they should be listed
```
Comment (willdurand) [inline apps/explorer/templates/explorer/pixelset_list.html]: s/they should be/they are

File: apps/explorer/tests/test_views.py [hunk 3]
```diff
@@ -880,16 +880,170 @@ def test_displays_message_after_redirect_when_invalid(self):
...
+            follow=True
+        )
+
+        self.assertEqual(response.status_code, 200)
+
+        self.assertContains(
+            response,
+            (
+                '<div class="message success">'
+                '2 pixelset(s) have been saved for exportation'
```
Comment (willdurand) [inline apps/explorer/tests/test_views.py]: s/exportation/export

File: apps/explorer/forms.py (hunk 1 above)
Comment (jmaupetit) [reply apps/explorer/forms.py]: It's not required. I choose to declare a new form in case we need to adapt it in the future.

Comment (willdurand) [reply apps/explorer/forms.py]: Do we still need the other one then?

**Pull request merged**